MSAL/src/native_auth/public/MSALNativeAuthPublicClientApplication.swift

```diff
@@ -102,11 +109,13 @@ public final class MSALNativeAuthPublicClientApplication: MSALPublicClientApplic
     init(
         controllerFactory: MSALNativeAuthControllerBuildable,
         inputValidator: MSALNativeAuthInputValidating,
-        internalChallengeTypes: [MSALNativeAuthInternalChallengeType]
+        internalChallengeTypes: [MSALNativeAuthInternalChallengeType],
```

Comment:
nit: would be good to have the challenge type list as last method parameter

Reply:
Good point thanks. I reordered the parameters